source/releasenotes/api-changes.rst

```diff
@@ -13,9 +13,14 @@
    specific language governing permissions and limitations
    under the License.
 
-
 API Changes Introduced in |release|
 ===================================
+For the complete list of API commands and params consult the `CloudStack Apidocs`_.
+
+The 4.12 API commands stayed as the 4.11.
```

Comment:
Can you reconfirm? I think there is script to validate this.

Reply:
@rhtyd I used the project https://github.com/pdion891/acs-api-commands to generate the API diff.
Diff 4.10.0 -> 4.11.2.0:
`$head --lines=15 acs-api-commands/diff-4100-41120/diff.txt `
```
Added commands:

    provisionCertificate (Issues and propagates client certificate on a connected host/agent using configured CA plugin)

    listElastistorPool (Lists the pools of elastistor)

    deleteServicePackageOffering (Delete Service Package)

    listAnnotations (Lists annotations.)

    enableHAForZone (Enables HA for a zone)

    enableHAForCluster (Enables HA cluster-wide)

    listNuageVspDomainTemplates (Lists Nuage VSP domain templates)

```
Diff 4.11.2.0 -> 4.12.0.0
`$cat acs-api-commands/diff-41120-41200/diff.txt`
```
Added commands:

Removed commands:

Changes in command type (sync versus async)


Changes in commands arguments:
```